setup.py

```diff
@@ -2222,7 +2222,18 @@ def maybe_fixup_exes():
             "user_access_control": "auto",
         },
     },
-    scripts=["pywin32_postinstall.py", "pywin32_testall.py"],
+    # This adds the scripts under Python3XX/Scripts, but doesn't actually do much
+    scripts=[
+        "win32/scripts/pywin32_postinstall.py",
+        "win32/scripts/pywin32_testall.py",
+    ],
```

Comment:
I'm keeping this for now as to not break some poor sysadmin's workflow of looking in Python's Scripts folder 😉  (then again, they were probably using .exe installers, but it's barely 4 lines to keep the support)